### ISSUE TITLE
Electron prebuilt is being renamed to electron

### ISSUE THREAD

**Issue by MarshallOfSound** [electron/electron-rebuild#91]
The `electron-prebuilt` module will be known as `electron` soon and as such this module needs to be updated to search for `electron` as well when determining the version.

The current workaround is to do `-e=node_modules/electron`

**Comment by zeke** [comment]
Thanks for catching this, @MarshallOfSound 

cc @kevinsawicki @jlord @zcbenz

**Comment by zaggino** [comment]
@zeke Using `electron-rebuild@1.2.0` I still have to use `-e=node_modules/electron` to get it working on `Windows_NT 10.0.14393, node v6.4.0, npm  v3.10.7`

**Comment by MarshallOfSound** [comment]
@zaggino I don't understand how you can have `electron-rebuild@1.2.0` as our current package.json is only up to `1.1.4`

https://github.com/electron/electron-rebuild/blob/master/package.json#L3

Did someone publish something without pushing to GitHub @zeke  ? 😆

**Comment by zaggino** [comment]
```
$ npm view electron-rebuild@* version
electron-rebuild@0.1.0 '0.1.0'
electron-rebuild@0.1.1 '0.1.1'
electron-rebuild@0.1.2 '0.1.2'
electron-rebuild@0.1.3 '0.1.3'
electron-rebuild@0.1.4 '0.1.4'
electron-rebuild@0.2.0 '0.2.0'
electron-rebuild@0.2.1 '0.2.1'
electron-rebuild@0.2.2 '0.2.2'
electron-rebuild@0.2.3 '0.2.3'
electron-rebuild@0.2.4 '0.2.4'
electron-rebuild@0.2.5 '0.2.5'
electron-rebuild@1.0.0 '1.0.0'
electron-rebuild@1.0.1 '1.0.1'
electron-rebuild@1.0.2 '1.0.2'
electron-rebuild@1.1.0 '1.1.0'
electron-rebuild@1.1.1 '1.1.1'
electron-rebuild@1.1.2 '1.1.2'
electron-rebuild@1.1.3 '1.1.3'
electron-rebuild@1.1.4 '1.1.4'
electron-rebuild@1.1.5 '1.1.5'
electron-rebuild@1.2.0 '1.2.0'
```

**Comment by Haumed** [comment]
Thanks for the `-e=node_modules/electron` fix! Any news on when this will hit npm? `1.2.0` def does nothing.

**Comment by zeke** [comment]
@Haumed which fix are you referring to? Can you share a commit or PR? If something on master is unpublished we can definitely cut a new release.

**Comment by Haumed** [comment]
I'm referring to PR #92. I did `npm update electron-rebuild` to get to `electron-rebuild@1.2.0` which supposedly closed this issue. But when I run `electron-rebuild --pre-gyp-fix --force` with only `electron` installed (not `electron-prebuilt`,)  the same error happens as on the previous version of  `electron-rebuild`:

```
Unable to find electron-prebuilt's version number, either install it or specify
an explicit version

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\e\\node_modules\\npm\\
in\\npm-cli.js" "run" "rebuild"
npm ERR! node v6.3.0
npm ERR! npm  v3.10.6
npm ERR! code ELIFECYCLE
npm ERR! AppName@0.1.6 rebuild: `electron-rebuild --pre-gyp-fix --force`
npm ERR! Exit status 4294967295
npm ERR!
npm ERR! Failed at the AppName@0.1.6 rebuild script 'electron-rebuild --pre-gyp
fix --force'.

...
```

This is fixed by appending `-e=node_modules/electron` to the command. However, I tried the same thing with only `electron-prebuilt-compiled` installed (vs `electron` or `electron-prebuilt`) and it gave a similar error without the `-e=node_modules/electron-prebuilt-compiled`):

```
Couldn't find electron-prebuilt and no --node-module-version parameter set, alwa
ys rebuilding
```

and the original error with `-e=node_modules/electron-prebuilt-compiled` attached:
```
Unable to find electron-prebuilt's version number, either install it or specify
an explicit version
```

I'm assuming that #92 would fix the second issue although I'm not sure as `electron-prebuilt-compiled` support is not new.

**Comment by MarshallOfSound** [comment]
@zeke It looks like `1.2.0` doesn't have the latest code changes in it.

<img width="322" alt="screen shot 2016-09-01 at 12 38 06 pm" src="https://cloud.githubusercontent.com/assets/6634592/18153485/1a0f6514-7041-11e6-8f90-03407f51ea10.png">

The electron locater JS file is not there

**Comment by zeke** [comment]
I just published 1.2.1. Please give that a try and let me know if it resolves the issue.

**Comment by zaggino** [comment]
Worked for me @zeke https://github.com/zaggino/brackets-electron/commit/d367d034030779854433e73b826712248a9303b2
